node-require/calculate.js

```diff
@@ -0,0 +1,25 @@
+const add = require('./add');
+const subtract = require('./subtract');
+const multiply = require('./multiply');
+const divide = require('./divide');
+
+let result;
+const num1 = parseInt(process.argv[2]);
+const num2 = parseInt(process.argv[4]);
```

Comment:
`calculate.js` should produce accurate outputs if the user supplies numbers with decimal places.